index.html

```diff
@@ -10,228 +10,307 @@
 </head>
 
 <body>
-     <!-- page 1 -->
+    <!-- page 1 -->
     <header>
-        <div class="logo-placeholder">
-            <a href="#" class="logo">Ouma NJ</a>
-        </div>
-        <nav class="nav-links">
-            <ul class="menu-bar">
-                <li class="hamburger-button">
-                    <img src="./images/icon-Menu.svg" alt="Menu button" class="menu-button" id="hamburger"/>
-                </li>
-            </ul>
+        <a class="logo" href="#">AIMA NJ</a>
+        <ul class="menu-bar">
+            <li class="hamburger-button">
+                <img src="./images/icon-Menu.svg" alt="Menu button" class="menu-button" id="hamburger"/>
+            </li>
+        </ul>
+        
+        <nav class="navigation">
+            <a class="links" href="#portfolio">Portfolio</a>
+            <a class="links" href="#about">About</a>
+            <a class="links" href="#contact">Contact</a>
         </nav>
     </header>
```

Comment:
This size of the logo text font used in your design is `22px` while in Figma used is `18px`. Kindly fix this. The comparison is as follows

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221279911-d1c7e07a-a8d3-44e4-8471-89f6d0319e0d.png)| ![image](https://user-images.githubusercontent.com/61361037/221279832-06d9b03a-a083-4ad8-bac3-a82dac8489e1.png)|